apps/src/p5lab/spritelab/commands/actionCommands.js

```diff
@@ -107,6 +109,14 @@ export const commands = {
     sprites.forEach(sprite => this.p5.edges.displace(sprite));
   },
 
+  edgesCollide(spriteArg) {
```

Comment:
mind adding a comment explaining what this does?

Reply:
Sure, I can do that. Unfortunately not of the other commands in this directory are well commented at this time.